apps/remix-ide/src/app/tabs/runTab/contractDropdown.js

```diff
@@ -300,13 +301,16 @@ class ContractDropdownUI {
       if (error) {
         return this.logCallback(error)
       }
-
+      _paq.push(['trackEvent', 'udapp', 'txTo', this.networkName])
       self.event.trigger('newContractInstanceAdded', [contractObject, address, contractObject.name])
 
       const data = self.runView.compilersArtefacts.getCompilerAbstract(contractObject.contract.file)
       self.runView.compilersArtefacts.addResolvedContract(helper.addressToString(address), data)
       if (self.ipfsCheckedState) {
+        _paq.push(['trackEvent', 'udapp', 'ipfsPublishChecked'])
```

Comment:
maybe this should also include the current network

Reply:
ok